lib/espejote.libsonnet

```diff
@@ -65,8 +66,195 @@ local managedResource(name, namespace) = {
   },
 };
 
+
+local generateRolesForManagedResource(manifest) =
+  local manifestMeta = std.get(manifest, 'metadata', {});
+  assert std.get(manifestMeta, 'name') != null : 'name is required';
```

Comment:
afaict all the `std.get(obj, field) != null` could be replaced with `std.objectHas(obj, field)` unless we care about the case where the field is explicitly set to `null`